deflect/qt/QmlStreamer.cpp

```diff
@@ -66,6 +66,11 @@ QQmlEngine* QmlStreamer::getQmlEngine()
     return _impl->getQmlEngine();
 }
 
+Stream *deflect::qt::QmlStreamer::getStream()
```

Comment:
Check:

Launcher is another process. I am not doing through the same process. If you guys know how to do it from another process without using a connection, please let me know.

https://github.com/BlueBrain/DisplayCluster/pull/264/files
FileBrowser.cpp